Dockerfile

```diff
@@ -0,0 +1,46 @@
+FROM python:3.9-slim as downloader
```

Comment:
```suggestion
FROM python:3.11-slim as downloader
```

Python 3.9 will be deprecated next year, so it is better to stick to modern version.